Neos.Neos/webpack.js.config.js

```diff
@@ -0,0 +1,55 @@
+const debug = process.env.NODE_ENV !== "production";
+const TerserPlugin = require("terser-webpack-plugin");
+const webpack = require("webpack");
+
+const javascriptConfig = {
+	context: __dirname,
+	devtool: debug ? 'source-map' : false,
+	entry: {
+		Main: ["./Resources/Public/JavaScript/index.js"],
+	},
+	output: {
+		path: __dirname + "/Resources/Public/JavaScript",
+		filename: "[name].min.js",
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				use: {
+					loader: "babel-loader",
+				},
+			},
+		],
+	},
+	plugins: [
+		new webpack.ProvidePlugin({
+			$: "jquery",
+			jQuery: "jquery",
```

Comment:
Where do we use jQuery?
Can't find it.

Reply:
We don't use it in the new code but it is used in the old bundle and the media ui uses all shapes of JavaScript ;)